CONTRIBUTING.md

```diff
@@ -160,6 +160,15 @@ update /etc/hosts
 127.0.0.1 mesherylocal.layer5.io
 ```
 
+### UI Lint Rules 
+
+We are using ES-Lint to maintain code quality & consistency in our UI Code. To make sure your PR passes all the UI & ES-Lint Tests, please see below :
+
+- Remember to run `make run-ui-lint` & `make run-provider-ui-lint` if you are making changes in Meshery-UI & Provider-UI respectively.
+- The above commands will only fix some basic indenting rules for you, you will have to manually check your code to ensure there is no duplications, un-used variables or un-declared constants.
```

Comment:
Will `eslint` flag and error out on these, too? Do this leave a gap where contributors may be frustrated by not knowing that their changes will fail a build check?

Reply:
Yes the flag will point all the errors, its just that some error will be fixed automatically but some won't be.

Reply:
Good deal.